_scss/_landing.scss

```diff
@@ -105,6 +105,11 @@ body#landing {
       height: 50px;
       line-height: 24px;
       padding: 13px 20px;
+
+      &:focus {
+        box-shadow: none;
+        outline: none;
+      }
```

Comment:
Looks like this doesn't _fully_ hide the "focus" (border still turns "blue" on the "light" version of the website); didn't search if there's other options to fully disable it nowadays.

Reply:
We don't have to complete hide the focus. The current version looks good.

Reply:
Yeah, I wanted to at least get rid of the "glow", as it distracted from the overall layout on the page